pkgs/development/libraries/libgourou/default.nix

```
@@ -0,0 +1,48 @@
+{ lib, stdenv, fetchzip, pugixml, updfparser, curl, openssl, libzip  }:
+
+stdenv.mkDerivation rec {
+  name = "libgourou";
+  version = "0.8.1";
+
+  src = fetchzip {
+    url = "https://indefero.soutade.fr/p/${name}/source/download/v${version}/";
+    sha256 = "sha256-X56K4z1+k62Q9pxnN8kx35oc7+uJJN/o/tpnKx1FjwE=";
+    extension = "zip";
+  };
+
+  postPatch = ''
+    patchShebangs scripts/setup.sh
+  '';
+
+  configurePhase = ''
+    mkdir lib
+    ln -s ${updfparser}/lib lib/updfparser
+  '';
```

Does something like this works instead?
```suggestion
  makeFlags = [
    "UPDFPARSERLIB=${updfparser}/lib"
  ];
```

---

No it doesn't.